pkg/rpcclient/wsclient.go

```diff
@@ -237,13 +375,57 @@ readloop:
 					break readloop
 				}
 			}
-			ok := make(map[chan<- Notification]bool)
+			ok := make(map[interface{}]bool)
+			if event == neorpc.MissedEventID {
+				c.subscriptionsLock.Lock()
+				for _, rcvr := range c.subscriptions {
+					ch := rcvr.Receiver()
+					rcvr.Close()
+					if ch != nil && !ok[ch] {
```

Comment:
@roman-khimov, I just can't make it prettier, maybe you have some thoughts about implementation improvement? We have to keep information about channels that were already closed in order not to close them one more time and not to send something else to them.